### ISSUE TITLE
Avoid repo clone to deploy Lynx

### ISSUE THREAD

**Issue by uvulpos** [Clivern/Lynx#61]
Depends on: #60 , #59 (optional)

As a user of Lynx, I just want to deploy this application via portainer or helm and don't clone the code locally.
Therefore I need an updated Readme File to simplify the deployment process

**Comment by Clivern** [comment]
No need to clone now to run on docker or k8s. there is another ticket to provide a helm chart.
soon I will provide a yaml definition to deploy on k8s
